projectroles/migrations/0024_project_star_app_setting.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations
+
+
+def migrate_stars(apps, schema_editor):
+    """Set rank values for existing roles"""
+    ProjectUserTag = apps.get_model('projectroles', 'ProjectUserTag')
+    Project = apps.get_model('projectroles', 'Project')
+    AppSetting = apps.get_model('projectroles', 'AppSetting')
+    for project in Project.objects.all():
+        user_tag = ProjectUserTag.objects.filter(project=project).first()
+        if user_tag is not None:
+            setting = AppSetting()
+            setting.project = project
+            setting.user = user_tag.user
+            setting.value = True
+            setting.name = 'project_star'
+            setting.user_modifiable = True
```

Comment:
`user_modifiable` is not relevant for the scope of this setting, so we don't need to define it here and it can simply remain default.